frontend/src/pages/AccountForm.tsx

```diff
@@ -134,22 +135,28 @@ function AccountForm(props: any) {
                 Major:
                 {currentUser.major}
               </Typography>
-              <Button onClick={handleEditMajor} startIcon={<EditIcon />} />
+              <Tooltip title="Edit Major">
+                <Button onClick={handleEditMajor} startIcon={<EditIcon />} />
+              </Tooltip>
             </ListItem>
           )}
         <ListItem>
           <Typography variant="subtitle1">Courses:</Typography>
```

Comment:
Less perhaps in this refactor's scope, but what should Courses be able to do for us? I haven't seen it used particularly in my profile, and there are no options to add or delete any courses from my list either

Reply:
Other than that, LGTM